<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Road数量计算工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>931</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label_prjname">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjpath">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjpath_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>110</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjname_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjname_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjpath_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjname_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjpath_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjpath_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>240</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjname_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjname_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjpath_6">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjname_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_prjpath_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>320</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>项目路径</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjname_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjpath_1">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjname_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjpath_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>100</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjname_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjpath_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjname_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjpath_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjname_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjpath_5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>240</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjpath_6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>280</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjname_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjpath_7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>481</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prjname_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_openprj"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="actionHintshuju"/>
    <addaction name="actionEiCAD"/>
    <addaction name="action_BIM"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>数量统计</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHintshuju">
   <property name="text">
    <string>Hint数据</string>
   </property>
  </action>
  <action name="actionEiCAD">
   <property name="text">
    <string>EiCAD数据</string>
   </property>
  </action>
  <action name="action_BIM">
   <property name="text">
    <string>公路BIM数据</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>边坡</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>排水沟</string>
   </property>
  </action>
  <action name="action_openprj">
   <property name="text">
    <string>打开项目</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
